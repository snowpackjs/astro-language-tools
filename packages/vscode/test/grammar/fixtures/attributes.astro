<!-- Test special characters inside attributes names -->

<div @class></div>
<div class_name></div>
<div class.astro></div>
