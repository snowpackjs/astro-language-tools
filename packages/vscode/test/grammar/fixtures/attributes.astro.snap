><!-- Test special characters inside attributes names -->
#^^^^ source.astro comment.block.astro punctuation.definition.comment.astro
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.astro comment.block.astro
#                                                     ^^^ source.astro comment.block.astro punctuation.definition.comment.astro
>
><div @class></div>
#^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro entity.name.tag.astro
#    ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro
#     ^^^^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro meta.attribute.@class.astro entity.other.attribute-name.astro
#           ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#            ^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#              ^^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro entity.name.tag.astro
#                 ^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.end.astro
><div class_name></div>
#^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro entity.name.tag.astro
#    ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro
#     ^^^^^^^^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro meta.attribute.class_name.astro entity.other.attribute-name.astro
#               ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#                ^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#                  ^^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro entity.name.tag.astro
#                     ^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.end.astro
><div class.astro></div>
#^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro entity.name.tag.astro
#    ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro
#     ^^^^^^^^^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro meta.attribute.class.astro.astro entity.other.attribute-name.astro
#                ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#                 ^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#                   ^^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro entity.name.tag.astro
#                      ^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.end.astro
><div i18nReady></div>
#^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro entity.name.tag.astro
#    ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro
#     ^^^^^^^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro meta.attribute.i18nReady.astro entity.other.attribute-name.astro
#              ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#               ^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#                 ^^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro entity.name.tag.astro
#                    ^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>